client/index.html

```diff
@@ -43,7 +43,22 @@
 		</footer>
 		<div id="main">
 			<div id="windows">
-				<div id="chat"></div>
+				<div id="chat">
+					<div class="window" style="display:block">
+						<div class="container">
+							<div class="row">
+								<div class="col-xs-12">
+									<h2 class="title">The Lounge is loading…</h2>
+								</div>
+								<div class="col-xs-12">
+									<p id="loading-page-message">Loading the app… <a href="http://enable-javascript.com/" target="_blank">Make sure to have JavaScript enabled.</a></p>
+									<p id="loading-slow" style="display:none">This is taking longer than it should, there might be connectivity issues.</p>
+								</div>
+							</div>
+						</div>
+					</div>
+					<script async src="js/loader.js"></script>
```

Comment:
Would call it loading-slow-check.js or something like that, unless you are planning to add more loader business there.